docs/models.md

```diff
@@ -0,0 +1,80 @@
+## Models
+
+Model_Name            | Description
+----------------------|------------------------------------------------------------------------------------------------------
+Subject               | Used to store *subject_name* and its corresponding *paper_id*.
+Teacher               | Used to store *teacher_name* and their *department*.
+TeacherSubjectMapping | Used to store the realationship between Subject and Teacher model. Eg. X teacher teach Y subject.
+Section               | It store the seciton, branch, year, shift and Teacher -----> Subject record of a particular class.
+Feedback	      | Used to store feedback of every teacher on the basis of the subject he/she teach.
+
+## Model's Field
+
+**1. Subject**
+
+Field_Name       | Description
+-----------------|--------------------------------------------------------------------
+**subject_name** | This field is used to store all subject name in the databse.
```

Comment:
Also, add one more parameter for `id` that django implements by default. Because documentation must be complete.